src/main/java/com/mycompany/cajeroautomatico/CajeroAutomatico.java

```diff
@@ -1,187 +0,0 @@
-
-package com.mycompany.cajeroautomatico;
-
-import java.io.IOException;
-
-public class CajeroAutomatico {
-
-    public static void main(String[] args) throws IOException {
-        Consola consola = new Consola();
-        Cliente cliente = new Cliente();
-        CC cc = new CC ();
-        CAPesos ca = new CAPesos();
-        CADolares dol = new CADolares();
-        Movimiento movimiento = new Movimiento();
-        LectorDeArchivos lector = new LectorDeArchivos();
-        
-        //Bienvenida y menu principal
-        String numTarjeta = consola.bienvenida();
-        String pin =  consola.pin();
-        cliente.validacion(numTarjeta, pin);
-        for (Cuenta cuenta: cliente.getMisCuentas()){
-            if(cuenta.getTipo() == 1){
-                ca.setTipo(cuenta.getTipo());
-                ca.setSaldo(cuenta.getSaldo());
-                ca.setAlias(cuenta.getAlias());
-            }if(cuenta.getTipo() == 2){
-                cc.setTipo(cuenta.getTipo());
-                cc.setSaldo(cuenta.getSaldo());
-                cc.setAlias(cuenta.getAlias());
-                cc.setDescubierto(cuenta.getDescubierto());
-            }if(cuenta.getTipo() == 3){
-                dol.setTipo(cuenta.getTipo());
-                dol.setSaldo(cuenta.getSaldo());
-                dol.setAlias(cuenta.getAlias());
-            }        
-        }
-        int num = consola.menuPrincipal();
-        //Consultas
-        if (num ==1 ){
-            num = consola.consultas();
-            if (num==1){
-                cliente.consultarMovimientos(cliente.getAlias());
-                movimiento.guardarMovimientos(cliente.getAlias(), 0, "Consulta de movimientos");
-            }if (num==2){
-                System.out.println("Su alias es: " + cliente.getAlias());
-                movimiento.guardarMovimientos(cliente.getAlias(), 0, "Consulta de alias");
-            }else {
-                System.out.println("Introdujo un numero incorrecto");
```

Comment:
throw new Exception...


https://rollbar.com/guides/java/how-to-throw-exceptions-in-java/#:~:text=Using%20the%20Throws%20keyword&text=We%20can%20throw%20either%20checked,abnormal%20termination%20of%20the%20program.